docs/user/guides/custom-domains.rst

```diff
@@ -72,7 +72,9 @@ For both |org_brand| and |com_brand|, HSTS and other custom headers can be set u
 We always return the HSTS header with a max-age of at least one year
 for our own domains including ``*.readthedocs.io``, ``*.readthedocs-hosted.com``, ``readthedocs.org`` and ``readthedocs.com``.
 
-Please contact :doc:`/support` if you want to add a custom header to your domain.
+.. sealso:
```

Comment:
This feels like an odd seealso, since it's not content. I think we want a note or tip? 

Also it's currently rendering as a comment because it's missing a `:` :D

Reply:
Definitely looks better as a note :+1: